### PR TITLE
daemon/logger/jsonfilelog: avoid some allocations

### PR DESCRIPTION
**Pull request by unclejack**
This PR makes jsonfilelog avoid some needless memory allocations.

### CODE REVIEW

**Comment by unclejack** [comment]
ping @LK4D4

File: daemon/logger/jsonfilelog/jsonfilelog.go [hunk 1]
```diff
@@ -33,7 +34,12 @@ func New(filename string) (logger.Logger, error) {
 func (l *JSONFileLogger) Log(msg *logger.Message) error {
 	l.mu.Lock()
 	defer l.mu.Unlock()
-	err := (&jsonlog.JSONLog{Log: string(msg.Line) + "\n", Stream: msg.Source, Created: msg.Timestamp}).MarshalJSONBuf(l.buf)
+	timestamp, err := timeutils.FastMarshalJSON(msg.Timestamp)
+	if err != nil {
+		return err
+	}
+	line := append(msg.Line, '\n')
```
Comment (LK4D4) [inline daemon/logger/jsonfilelog/jsonfilelog.go]: You can use append right in line below without creating variable. Don't know if it's worth it.

Comment (unclejack) [inline daemon/logger/jsonfilelog/jsonfilelog.go]: @LK4D4 I've made the change. I guess avoiding it doesn't hurt that much.

**Comment by LK4D4** [comment]
LGTM

**Comment by LK4D4** [comment]
ping @jfrazelle @tiborvass

**Comment by tiborvass** [comment]
LGTM

**Comment by thaJeztah** [comment]
Is this something for the 1.6 release?

**Comment by jfrazelle** [comment]
nope too late in the game for new drivers in 1.6 but will be in 1.7 ;)

**Comment by thaJeztah** [comment]
Fair, just checking, in case it was important, thanks!

**Pull request merged**